packages/apollo-server-integration-testsuite/src/index.ts

```diff
@@ -620,7 +620,7 @@ export default (createApp: CreateAppFunc, destroyApp?: DestroyAppFunc) => {
         app = await createApp();
         const req = request(app)
           .post('/graphql')
-          .send({ query: introspectionQuery });
+          .send({ query: getIntrospectionQuery() });
```

Comment:
The `introspectionQuery` constant is deprecated in `graphql@15` after https://github.com/graphql/graphql-js/pull/2124.  The `getIntrospectionQuery` method has been available since `graphql@0.12.x`, so our reliance on it now should not be a problem.